CHANGELOG.md

```diff
@@ -36,8 +36,19 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ## Unreleased
+
+- [#670](https://github.com/babylonlabs-io/babylon/pull/670) add function to recover pub key from sig
+- [#646](https://github.com/babylonlabs-io/babylon/pull/646) backport: chore(ante): add tx gas limit decorator and local mempool config
+
+### Improvements
+
+- [#666](https://github.com/babylonlabs-io/babylon/pull/666) Upgrade to wasmvm v2.2.3.
+
 ### Bug fixes
 
+- [#664](https://github.com/babylonlabs-io/babylon/pull/664) crypto: fix adaptor sig timing side channels
+- [#663](https://github.com/babylonlabs-io/babylon/pull/653) crypto: fix adaptor sig validity and typos
+- [#662](https://github.com/babylonlabs-io/babylon/pull/662) crypto: check if Z==1 in ToBTCPK
```

Comment:
I thought the changelog shall not include backport PRs?

Reply:
Oh, I didn't knew that. Ran into conflicts when backporting, and resolved it this way. Do you want me to remove those then?

Reply:
ahhh thanks for cleaning it up! :) 

In general changelog should contain orginal pr's (the one based on `main`) as properly made backport will reference orignal pr, so just by looking at orignal pr you can see where it was backported.

Reply:
All right, I'll update the refs to the main ones.